.travis.yml

```diff
@@ -25,3 +27,17 @@ after_success:
   - ./scripts/build.sh
   - ./test/container.sh
   - ./scripts/push.sh
+
+before_deploy:
+  - eval "$(ssh-agent -s)"
+  - chmod 600 $TRAVIS_BUILD_DIR/${REPO}_deploy_rsa
+  - ssh-add $TRAVIS_BUILD_DIR/${REPO}_deploy_rsa
+addons:
+  ssh_known_hosts: $BASTION
+deploy:
+  provider: script
+  skip_cleanup: true
+  # The deploy.sh file points to the deploy file on the bastion. Not one in the repo.
+  script: ssh -o StrictHostKeyChecking=no $BASTION_USER@$BASTION ./deploy.sh ${REPO} $COMMIT
```

Comment:
If the `ssh_known_hosts` addon is being used, why do we need to `StrictHostKeyChecking=no` here?